modules/programs/zsh.nix

```diff
@@ -34,6 +34,11 @@ let
     vicmd = "bindkey -a";
   };
 
+  highlighters = {
+    zsh-fast-syntax-highlighting = "source ${pkgs.zsh-fast-syntax-highlighting}/share/zsh/site-functions/fast-syntax-highlighting.plugin.zsh";
+    zsh-syntax-highlighting = "source ${pkgs.zsh-syntax-highlighting}/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh";
+  };
```

Comment:
On second thought, this is way more annoying than I thought because of the additional filename difference. I think we should just go with what you had to start with. Sorry for the back and forth!